ui/routes/index.tsx

```diff
@@ -1,13 +1,13 @@
 import { Router } from '@reach/router'
 import React from 'react'
 import { ExampleRoute } from 'ui/routes/Example'
-import { IndexRoute } from 'ui/routes/Index'
+import { HomeRoute } from 'ui/routes/Home'
 import { QueryRoute } from 'ui/routes/Query'
 
 export function Routes() {
   return (
     <Router>
-      <IndexRoute path='/' />
+      <HomeRoute path='/home' />
```

Comment:
We still want a route on `/`, I think.

Reply:
When deploying from mac a `/` route doesn't seem to work.

Reply:
Even if you name it `Home`?

Reply:
Apparently not